gravitee-apim-console-webui/src/components/gio-side-nav/gio-side-nav.component.ts

```diff
@@ -172,6 +180,10 @@ export class GioSideNavComponent implements OnInit {
     return this.filterMenuByPermission(mainMenuItems);
   }
 
+  private getMenuItemIconRight$(licenseOptions: LicenseOptions) {
+    return this.gioLicenseService.isMissingFeature$(licenseOptions).pipe(map((notAllowed) => (notAllowed ? 'gio:lock' : null)));
+  }
+
```

Comment:
The same change must be done in https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-console-webui/src/management/api/api-ng-navigation/api-ng-v1-v2-menu.service.ts and https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-console-webui/src/management/api/api-ng-navigation/api-ng-v4-menu.service.ts in order to lock Alerts in API sub-menu